app/ui/src/app/dashboard/dashboard.component.spec.ts

```diff
@@ -55,15 +69,21 @@ describe('DashboardComponent', () => {
           SyndesisCommonModule
         ],
         declarations: [
+          DashboardMetricsComponent,
           DashboardComponent,
-          EmptyStateComponent,
+          DashboardEmptyComponent,
           DashboardConnectionsComponent,
           DashboardIntegrationsComponent
         ],
         providers: [
           ConfigService,
+          // {
```

Comment:
I guess this can go?  As well as the const configServiceStub up above?

Reply:
Certainly Sir!